app/models/billing/usage/tracker.rb

```diff
@@ -2,6 +2,9 @@ class Billing::Usage::Tracker < BulletTrain::Billing::Usage.base_class.constanti
   # e.g. `belongs_to :team`
   belongs_to BulletTrain::Billing::Usage.parent_association
 
+  # just for specs
+  scope :billable, -> { all }
```

Comment:
this is the only object I can create in the repo in order to test the `have` functionality, since we use `exist_count_for`